tiledb/cloud/group.py

```diff
@@ -0,0 +1,127 @@
+from typing import Dict, List
+
+from tiledb.cloud import client
+from tiledb.cloud import tiledb_cloud_error
+from tiledb.cloud import utils
+from tiledb.cloud.array import delete_array
+from tiledb.cloud.rest_api import ApiException as GenApiException
+
+# Re-export for compatibility.
+split_uri = utils.split_uri
+
+
+def get_group(uri):
+    """
+    Returns the cloud metadata
+
+    :return: metadata object
+    """
+    (namespace, group_name) = split_uri(uri)
+    api_instance = client.client.groups_api
+
+    try:
+        return api_instance.get_group(namespace, group_name, async_req=async_req)
```

Comment:
`async_req` is never defined here; it's false by default so you can just remove it entirely:
```suggestion
        return api_instance.get_group(namespace, group_name)
```